tutorials/cdn-fastly.md

```diff
@@ -174,14 +176,23 @@ calls back to your Cloud Storage bucket. And by using an Origin
 Shield that is connected to Google's network edge, you can increase
 the speed of the shield's origin pulls.
 
+It is normally recommended to use a multi-regional Cloud Storage
+[bucket location](https://cloud.google.com/storage/docs/locations), but when
+using an Origin, it is instead recomended to use a regional or dual-regional
+location. This ensures the data is always close to the Orign, improving
```

Comment:
This ensures the data is always close to the Orign, -> This ensures the data is always close to the origin,